bazel/linux/qemu.bzl

```diff
@@ -127,3 +160,49 @@ See the RuntimeBundleInfo provider for details.
         }
     ),
 )
+
+kernel_qemu_aarch64_run = rule(
+    doc = """Runs code in a aarch64 qemu instance.
+
+The code to run is specified by using the "runner" attribute, which
+pretty much provides a self contained directory with an init script.
+See the RuntimeBundleInfo provider for details.
+
+Note to change the kernel, initramfs, kernel cmdline etc. a new FIT image with
+necessary changes has to be created and its KernelImageInfo passed as kernel_image
+to this rule
+""",
+    implementation = _kernel_qemu_run,
+    executable = True,
+    # Note to change the kernel, initramfs, kernel cmdline etc. a new FIT image with
+    # necessary changes has to be created and its KernelImageInfo passed as kernel_image
+    # to this rule
+    attrs = dict(
+        create_runner_attrs(
+            template_init_default = Label("//bazel/linux:templates/init-qemu.template.sh"),
+        ),
+        **{
+            "archs": attr.string_list(
+                default = ["aarch64"],
```

Comment:
if we are passing the archs, do we need a separate kerenl_qemu_run rule ? Could we not just use the existing one ?